sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -467,7 +467,15 @@
    * @since 4.0
    */
   String WORKING_DIRECTORY = "sonar.working.directory";
+  
   String WORKING_DIRECTORY_DEFAULT_VALUE = ".sonar";
+  
+  /**
+   * @since 5.2
+   */
+  String GLOBAL_WORKING_DIRECTORY = "sonar.globalWorking.directory";
```

Comment:
Should check with @simonbrandhof what is the "official" format for properties since he already reported that we are not consistent.